lib/nexpose/api_request.rb

```diff
@@ -97,27 +97,31 @@ def execute(options = {})
             end
           end
         end
-        # This is a hack to handle corner cases where a heavily loaded Nexpose instance
-        # drops our HTTP connection before processing. We try 5 times to establish a
-        # connection in these situations. The actual exception occurs in the Ruby
-        # http library, which is why we use such generic error classes.
-      rescue OpenSSL::SSL::SSLError => e
+      # This is a hack to handle corner cases where a heavily loaded Nexpose instance
+      # drops our HTTP connection before processing. We try 5 times to establish a
+      # connection in these situations. The actual exception occurs in the Ruby
+      # http library, which is why we use such generic error classes.
+      rescue OpenSSL::SSL::SSLError
         if @conn_tries < 5
           @conn_tries += 1
           retry
         end
-      rescue ::ArgumentError, ::NoMethodError => e
+      rescue ::ArgumentError, ::NoMethodError
         if @conn_tries < 5
           @conn_tries += 1
           retry
         end
       rescue ::Timeout::Error
+        $stdout.puts @http.read_timeout
```

Comment:
I think this is a debugging statement that needs to be removed.

Reply:
yeah it will be, i'm still testing this branch